### ISSUE TITLE
Unknown import symbol

### ISSUE THREAD

**Issue by tigerjack** [microsoft/pyright#1207]
**Describe the bug**
error: "Program" is unknown import symbol (reportGeneralTypeIssues)

**To Reproduce**

1. `pyenv install 3.6.12`
2. `pyenv virtualenv 3.6.12 myqlm`
3. `pyenv activate myqlm`
4. `pip install myqlm`
5. `echo "from qat.lang.AQASM import Program" > a.py`
6. `pyright --verbose a.py`

Output is 
```
No configuration file found.
Search paths found for configured python interpreter:
  /home/simone/.pyenv/versions/3.6.12/lib/python3.6
  /home/simone/.pyenv/versions/3.6.12/lib/python3.6/lib-dynload
  /home/simone/.pyenv/versions/myqlm/lib/python3.6/site-packages
When attempting to get search paths from python interpreter:
  Executing interpreter: 'python3'
  Skipping '/home/simone/.pyenv/versions/3.6.12/lib/python36.zip' because it is not a valid directory
  Received 3 paths from interpreter
    /home/simone/.pyenv/versions/3.6.12/lib/python3.6
    /home/simone/.pyenv/versions/3.6.12/lib/python3.6/lib-dynload
    /home/simone/.pyenv/versions/myqlm/lib/python3.6/site-packages
stubPath /home/simone/typings is not a valid directory.
Assuming Python platform Linux
Searching for source files
Found 1 source file
/home/simone/a.py
  1:28 - error: "Program" is unknown import symbol (reportGeneralTypeIssues)
1 error, 0 warnings, 0 infos
Completed in 0.424sec
```

**Expected behavior**
The import should be correctly resolved.

**Screenshots or Code**
As before
```python
from qat.lang.AQASM import Program
```

**VS Code extension or command-line**
command line tool

**Comment by erictraut** [comment]
I'm not able to install this package because it apparently isn't supported on MacOS, so I can't diagnose the problem directly.

This is almost definitely an issue with the package, not with Pyright. Does the package claim to be typed (i.e. does it include a "py.typed" file as specified in PEP 561)? If not, then it probably won't work well with a type checker like Pyright. You might need to find or create type stubs for the library.
